types/leaflet-areaselect/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for Leaflet.areaSelect 0.1
+// Project: https://github.com/heyman/leaflet-areaselect
+// Definitions by: André Wallat <https://github.com/awallat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="leaflet" />
+
+declare global {
```

Comment:
I removed the global declaration, but also removed those export statements since all the other leaflet-plugins are declared similar.
Is it ok? ;)